### PR TITLE
Rm vid proj

### PR DESCRIPTION
**Pull request by PatrikLundgren**
Added remove project to tree view and filehandler, added destructor to project.

### CODE REVIEW

File: ViAn/Filehandler/filehandler.cpp [hunk 1]
```diff
@@ -222,12 +226,19 @@ FH_ERROR FileHandler::delete_project(Project* proj){
...
+}
+/**
+ * @brief FileHandler::remove_video
+ * @param proj_id
+ * @param vid_id
+ */
+void FileHandler::remove_video(ID proj_id, ID vid_id){
+    Project* proj = this->get_project(proj_id);
+    proj->remove_video(vid_id);
 }
```
Comment (GeorgeYildiz) [inline ViAn/Filehandler/filehandler.cpp]: Although you find this method self-explanatory you should still comment what it does. A sentence is enough. Something like "Removes the search path to the video in the project" or something.

File: ViAn/Filehandler/project.cpp [hunk 2]
```diff
@@ -19,18 +20,40 @@ Project::Project(){
...
-* @param vid
-* add given video to project
-*/
-void Project::add_video(Video* vid)
-{
-    this->m_videos.push_back(vid);
+ * @brief Project::remove_video
+ * @param id
+ */
+void Project::remove_video(ID id){
```
Comment (GeorgeYildiz) [inline ViAn/Filehandler/project.cpp]: Here is another one that has the same function name in a different class. This is a reason why you need commenting.

File: ViAn/Filehandler/project.cpp [hunk 3]
```diff
@@ -73,9 +97,9 @@ ProjectStream& operator<<(ProjectStream &ps, const Project& proj){
     int vidcounter = proj.videos.size();
     ps.videos << vidcounter << " ";
     for(auto vid = proj.videos.rbegin(); vid != proj.videos.rend(); ++vid){
-        Video* v = *vid;
-        ps.videos << *v << " ";
-        vidcounter++;
+        Video* v = vid->second;
+            ps.videos << *v << " ";
+            vidcounter++;
     }
```
Comment (henpe071) [inline ViAn/Filehandler/project.cpp]: vidcounter -> snake_case
och varför är det en tab innan ps.videos << *v << " "; och vidcounter++; ?

Comment (PatrikLundgren) [inline ViAn/Filehandler/project.cpp]: rests from merge, will update now

**Pull request closed without merging**